domain.yml

```diff
@@ -40,7 +40,7 @@ intents:
 - contact_sales
```

Comment:
please remove the domain file from the diff, you didn't have to add anything to the domain, did you? if so, please add it manually so the diff only reflects those changes

Reply:
I added `utter_ask_docs_help` to the domain